src/schema/libraries.js

```diff
@@ -296,8 +300,7 @@ export const resolvers = {
       );
 
       // replace detailHoldings.holdingstatus with the merged holdings
-      detailedHoldings.holdingstatus = branchHolding;
```

Comment:
Denneher bug var grum. Svær at finde. Fordi detailedHoldings bliver muteret (som bliver ind idataloader-cache), så bliver holdings for de efterfølgende branches forkerte. Forhåbentligt får vi mange flere grønne lamper nu